tests/e2e/k8s/test_basic.go

```diff
@@ -43,7 +43,7 @@ func (s *TestSuite) TestConnectivityCRD() {
 			Name: httpEchoService.Name,
 			Port: 80,
 		}
-		require.Nil(s.T(), cl[1].CreateImportCRD(importedService, cl[0], httpEchoService.Name))
+		require.Nil(s.T(), cl[1].CreateImport(importedService, cl[0], httpEchoService.Name))
 
```

Comment:
You didn't want to combine the tests? with CRD or without?

Reply:
I will do it in a separate PR that enhances tests.